### ISSUE TITLE
Plugin does not exist.

### ISSUE THREAD

**Issue by CraXyOW3** [runfalk/certbot-loopia#4]
Get an error running certbot -a certbot-loopia:credentials
But running certbot -a certbot-loopia:auth is valid.

Also, how to predefine credentials.ini locations?

**Comment by runfalk** [comment]
I haven't tried the program in a while, but it should work like this:

```
certbot certonly --text --authenticator certbot-loopia:auth --certbot-loopia:auth-credentials credentials.ini -d www.your-domain.com
```

The `credentials.ini` should be set up as described in the README:

```
certbot_loopia:auth_user = user@loopiaapi
certbot_loopia:auth_password = passwordgoeshere
```

**Comment by coddan** [comment]
That did not work for me atleast.

**Comment by runfalk** [comment]
If you want help I need more information. What command are you trying to run and what is your error message?

**Comment by runfalk** [comment]
@CraXyOW3 could you try the updated instructions in the README? Note the added `sudo` for the commands.

**Comment by CraXyOW3** [comment]
If I recall correctly I did use sudo when I set it up with the info you gave in this issue.
After that, the script has been running flawlessly for a year with exactly 0 faults and 100% success rate.

Technically only like 4 times, but with no problems, due to letsencrypts 90 days intervall.

Did the same procedure for a friend and no problems there either.

I'd suggest @coddan should add more information of his problem.
